lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -658,6 +652,23 @@ private function completeIdGeneratorMapping(ClassMetadataInfo $class): void
         }
     }
 
+    private function determineIdGeneratorStrategy(AbstractPlatform $platform): int
+    {
+        if ($platform->getName() === 'oracle' || $platform->getName() === 'postgresql') {
```

Comment:
Oracle doesn't support identity columns, so it seems like there's no need to list it here.

Reply:
> Oracle doesn't support identity columns

That is correct. My intention here was to prevent ORM from changing behavior if one fine day Oracle starts to support identity columns.